### DISCUSSION TITLE
Android: Offline messages are not getting

### DISCUSSION THREAD

**Discussion by manishoctal** [processone/ejabberd#3682] [Questions] [answered]
We are using smack for using XMPP. when the user sends a message to offline users, then these messages are not getting once the user is online. 
I tried with following link: 
https://stackoverflow.com/a/36304076
but not working.. Any help?

**Comment by licaon-kter** [comment, marked as answer]
With any client (eg. Gajim or Conversations) or some custom one?

**Reply by manishoctal** [reply]
We are using our custom android application. Smack library.

**Reply by licaon-kter** [reply]
Maybe that's umm... not ready yet. Try other clients too :)

Do paste your full config on https://gist.guthub.com
